src/collective/cover/browser/configure.zcml

```diff
@@ -14,12 +14,12 @@
       />
   <browser:page
       name="content-search"
-      for="*"
+      for="..interfaces.ICover"
       permission="cmf.ModifyPortalContent"
       class=".contentchooser.ContentSearch"
       />
   <browser:pages
-      for="*"
+      for="Products.CMFCore.interfaces.IFolderish"
```

Comment:
``@@jsonByType`` view is used into contentchooser tree, and it list the available content into a folder (the folder is the context here)